### DISCUSSION TITLE
Change Placeholder Color

### DISCUSSION THREAD

**Discussion by developerinvnr** [wire-elements/spotlight#80] [Q&A]
![adv](https://user-images.githubusercontent.com/85278280/170429965-c78b8bae-ce42-449f-997b-424966132500.png)

in above screenshot ... the placeholder is not visible clearly.. how can i change its color

**Comment by PhiloNL** [comment]
My guess is that you are overriding the color with your own CSS. I would recommend inspecting the element with your browser inspector to see what is going on with the CSS.
